src/Checkout.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\ShopStandards\Checkout.
+ */
+
+namespace Netzstrategen\ShopStandards;
+
+/**
+ * Checkout related settings and actions.
+ */
+class Checkout {
+
+  public static function init() {
+    add_filter('woocommerce_get_settings_shop_standards', __CLASS__ . '::woocommerce_get_settings_shop_standards');
+
+    if (is_admin()) {
+      return;
+    }
+
+    // Add salutation field to checkout.
+    if (get_option(Plugin::PREFIX . '_add_salutation_field') === 'yes') {
+      add_filter('woocommerce_checkout_fields', __CLASS__ . '::woocommerce_checkout_fields');
+    }
+  }
+
+  /**
+   * Adds checkout specific settings.
+   *
+   * @implements woocommerce_get_settings_shop_standards
+   */
+  public static function woocommerce_get_settings_shop_standards(array $settings): array {
+    $settings = array_merge($settings, [
```

Comment:
It would be simpler to use regular array additions instead of a merge here:
```php
$settings[] = [
  'type' => 'title',
  'name' => ...
];
$settings[] = [
  'type' => 'checkbox',
  ...
];
```
I'd also recommend to always state the type first. Makes it easier to understand what each code block represents.